mesa/experimental/jupyter_viz.py

```diff
@@ -181,7 +181,7 @@ def on_value_play(change):
     def do_step():
         model.step()
         previous_step.value = current_step.value
-        current_step.value += 1
+        current_step.value = model.steps
```

Comment:
should this be `model.steps_` ?

Reply:
Yes, good catch. Almost broke JupyterViz with this line.